docs/private-dns/overview.md

```diff
@@ -58,7 +60,7 @@ Here is a simple comparison of features available in Public AdGuard DNS and Priv
 
 Every device that you add in the AdGuard DNS panel has its own unique address that can be used if the device supports modern encrypted DNS protocols (DoH, DoT, and DoQ).
 
-## Linked IP
+### For devices that do not support DoH, DoT, and DoQ
```

Comment:
Linking IP is not the only way, there's also dedicated IPv6 addresses which is a better option (but not everyone can use it).